src/components/repository/RepositoryViewBase/RepositoryViewBase.vue

```diff
@@ -23,6 +23,36 @@
             :repository="repository"
             class="rpstr-vw-bs__card__header__navigation" />
         </div>
+        <div class="rpstr-vw-bs__status-bar">
+          <div class="bh-grid">
+            <div class="bh-grid__item">
+              <div class="bh-grid bh-grid--fit-content rpstr-vw-bs__status-bar__align-center">
```

Comment:
Rename `rpstr-vw-bs__status-bar__align-center` to `rpstr-vw-bs__status-bar__authorization`